news/2020-11-15-osu-mapping-olympiad-7-concludes.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "osu! Mapping Olympiad #7: Results (osu!mania)"
+date: 2020-11-15 18:00:00 +0000
+---
+
+Here come the results of the 2nd edition of the osu!mania Mapping Olympiads, including sick surprises for sure! 
+
+![](https://osu.ppy.sh/wiki/shared/news/banners/osu_mapping_olympiad.jpg)
+
+For the first time in the osu!mania Mapping Olympiad's history, two Olympians receive the same score, and both reach the first place! Please give a huge round of applause to [**Kawawa**](https://osu.ppy.sh/users/4647754) and [**Toaph Daddy**](https://osu.ppy.sh/users/7616811)! 
+
+During this contest iteration, contestants had to make a full mapset on one of 5 songs from several artists, featuring [PUP](https://osu.ppy.sh/beatmaps/artists/79), [Hyun](https://osu.ppy.sh/beatmaps/artists/33), [IAHN](https://osu.ppy.sh/beatmaps/artists/3), [Frums](https://osu.ppy.sh/beatmaps/artists/68) and [Cranky](https://osu.ppy.sh/beatmaps/artists/23)! You can find more details about this contest's criteria over [here](https://osu.ppy.sh/community/contests/97)! 
+
+The leaderboards of the 10 best entries are shown right below!
+
+| Rank | Username      | Entry Name           | Rankability (45) | Song Interpretation (60) | Originality (45) | Hitsounding (15) | Judge Impression (30) | Total Score |
+|------|---------------|----------------------|------------------------|-----------------|----------------|-----------------|------------------|-------------|
+| #1   | Kawawa   | Moonflower        | 41.5                     | 42            | 27             | 13              | 17             | **140.5**     |
+| #1   | Toaph Daddy   | Zinnia        | 33                     | 46            | 30.5             | 10              | 21             | **140.5**     |
+| #3   | lenpai       | Lupine         | 35                     | 39              | 36           | 12            | 16               | **138**     |
+| #3   | BringoBrango       | Azalea         | 18                     | 49.5              | 34.5           | 13            | 23               | **138**     |
+| #5   | Garalulu | Lily       | 14                   | 41.5            | 30             | 12            | 18               | **115.5**   |
+| #6   | chouyaaa      | Hepatica     | 27.5                   | 31            | 19.5           | 13.5            | 13.5             | **105**   |
+| #7   | EimitMaggi999         | Iris      | 31                   | 27.5              | 18           | 13.5              | 12.5               | **102.5**     |
+| #8   | Noch Einen       | Tulip         | 35                     | 26              | 21           | 8.5              | 11.5               | **102**   |
+| #9   | fktang     | Buttercup        | 33                     | 29            | 14           | 14.5            | 10               | **100.5**   |
+| #10   | FAMoss   | Celosia     | 36.5                     | 22            | 14           | 14.3            | 7.5             | **94.3**     |
```

Comment:
```suggestion
| Rank | Username | Entry Name | Rankability (45) | Song Interpretation (60) | Originality (45) | Hitsounding (15) | Judge Impression (30) | Total Score |
| :-- | :-- | :-- | :-: | :-: | :-: | :-: | :-: | :-: |
| #1 | Kawawa | Moonflower | 41.5 | 42 | 27 | 13 | 17 | **140.5** |
| #1 | Toaph Daddy | Zinnia | 33 | 46 | 30.5 | 10 | 21 | **140.5** |
| #3 | lenpai | Lupine | 35 | 39 | 36 | 12 | 16 | **138** |
| #3 | BringoBrango | Azalea | 18 | 49.5 | 34.5 | 13 | 23 | **138** |
| #5 | Garalulu | Lily | 14 | 41.5 | 30 | 12 | 18 | **115.5** |
| #6 | chouyaaa | Hepatica | 27.5 | 31 | 19.5 | 13.5 | 13.5 | **105** |
| #7 | EimitMaggi999 | Iris | 31 | 27.5 | 18 | 13.5 | 12.5 | **102.5** |
| #8 | Noch Einen | Tulip | 35 | 26 | 21 | 8.5 | 11.5 | **102** |
| #9 | fktang | Buttercup | 33 | 29 | 14 | 14.5 | 10 | **100.5** |
| #10 | FAMoss | Celosia | 36.5 | 22 | 14 | 14.3 | 7.5 | **94.3** |
```
since we do not beautify the tables, correct formatting should be something like this

Reply:
Applied